lib/paperclip/storage/eitheror.rb

```diff
@@ -73,7 +73,7 @@ def usable_storage
         either_exists = @either.exists?
         or_exists = @or.exists?
 
-        return @either if either_exists || !or_exists
+        return @either if !or_exists || either_exists
```

Comment:
lol, you are absolutely correct :~

I actually meant to move the `@*.exists?` into line 76. I'll push that soon.